test/go.mod

```diff
@@ -2,9 +2,9 @@ module sigs.k8s.io/cluster-api-provider-vsphere/test
 
 go 1.22.0
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.9.0-rc.0
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.9.0-rc.1
 
-replace sigs.k8s.io/cluster-api/test => sigs.k8s.io/cluster-api/test v1.9.0-rc.0
+replace sigs.k8s.io/cluster-api/test => sigs.k8s.io/cluster-api/test v1.9.0-rc.1.0.20241204073105-63592b4d3c1c
```

Comment:
Q: why do we need .0.20241204073105-63592b4d3c1c here?

Reply:
Because v1.9.0-rc.1 does not yet have https://github.com/kubernetes-sigs/cluster-api/pull/11531